src/main/scala/gitbucket/core/service/IssuesService.scala

```diff
@@ -31,6 +31,9 @@ trait IssuesService {
       Issues filter (_.byPrimaryKey(owner, repository, issueId.toInt)) firstOption
     else None
 
+  def getAllIssues(owner: String, repository: String)(implicit s: Session): List[Issue] =
+    Issues filter (_.byRepository(owner, repository)) sortBy (_.issueId desc) list
```

Comment:
I think it would be more user-friendly if only open issues/PRs are visible, just like milestone filtering.
In many cases, there are hundreds of closed issues/PRs in a long-living repository.

![image](https://user-images.githubusercontent.com/17608272/90409652-6632ec00-e0e4-11ea-9e30-f961b72ac9c6.png)

Reply:
@SIkebe Thx for your review! Showing all Issues and PR is because of GitHub. They actually show everything. And I thought it make sense because jquery completion show only 10 issues and PR at once and ordered by issue id so from new to old. In many case new id is still open. And by contrary it could be inconvenient if closed issue can not be show up. What do you think?

Reply:
Or maybe we can change color by status (open or close)...

Reply:
>Showing all Issues and PR is because of GitHub. They actually show everything

GitHub shows only open issues/PRs. GitLab also does the same thing. In my experience, you refer an open issue more frequently than closed one.

![image](https://user-images.githubusercontent.com/17608272/90413783-e7d94880-e0e9-11ea-881c-254dd79657c7.png)

Reply:
GitLab

-----
![image](https://user-images.githubusercontent.com/17608272/90414730-1e639300-e0eb-11ea-8034-0ff031022d55.png)

Reply:
note: GitHub shows open issues/PRs ordered by id **ASC** not DESC. I'm not sure which one is better, though...

![image](https://user-images.githubusercontent.com/17608272/90415248-cb3e1000-e0eb-11ea-8798-f5a283749e50.png)

Reply:
@SIkebe Oh.. I mistook. Yeah. GitHub shows only opened one! But I also fount out if you type completely closed id, it shows up! I did not know...

Reply:
>But I also fount out if you type completely closed id, it shows up!

Hmm, I can not repro that.
![completion](https://user-images.githubusercontent.com/17608272/90418588-44d7fd00-e0f0-11ea-8e2a-5a427021858d.gif)

Reply:
!? So strange... Depends on browser... ?  I am using Chromium base Edge.

Reply:
Ah, not a browser specific behavior. GitHub seems to show all issues/PRs if its id is 1700 or later in this repo 😅

Reply:
So it means 
- newer ids -> show all
- older ids -> only opened one

little complicated query might be required... hmm.

Reply:
Yeah, IMHO, only open issues/PRs would be enough for GitBucket as with GitLab.

@takezoe What do you think?

Reply:
@SIkebe I'm considering about showing only opened Issue and PR since you suggested.  Yeah I think It might be better like that because I think the number case you mention opened one is many more than that with closed one.

Reply:
I also agree with showing only open issues/PR as completion proposals.

Reply:
@SIkebe @takezoe 
Ok! So I will fix PR to show only Opened issues/PRs. 
About this method ( `getAllIssues` ), I think it might be better preserved because it's more general than getting only opened isssues/PRs.
And filter at IssueController like fllowing code. 

What do you think?

```
          getAllIssues(repository.owner, repository.name)
            .filterNot(_.closed)
            .map { t =>
```

Reply:
In terms of performance, it would be better to narrow down at the SQL layer. You can implement `getAllIssues` any time you really need it.

Reply:
Yeah. It would be slow to get all... ok! So I will fix sql layer! I mean this method!

Reply:
@takezoe @SIkebe I fixed method.